02-oop-inheritance/Exercise 8 finished/js/EventEmitter.js

```diff
@@ -0,0 +1,20 @@
+export default class EventEmitter {
+
+    constructor() {
+        this.eventsMap = new Map();
+    }
+
+    on(event,instructions) {
+        if (!this.eventsMap.has(event)) {
+            this.eventsMap.set(event,instructions);
+        }
+    }
+
+    emit(event) {
+        return this.eventsMap.get(event); 
+        }
+    
+    off(event,instructions) {
+        this.eventsMap.delete(event,instructions);
```

Comment:
You only needs to provide a `key` as parameter to the `delete` method, the value is not needed.